hail/python/hailtop/hailctl/config/cli.py

```diff
@@ -42,29 +142,29 @@ def get_section_key_path(parameter: str) -> Tuple[str, str, Tuple[str, ...]]:
     sys.exit(1)
 
 
+def complete_config_variable(incomplete: str):
+    for name, help_text, _ in config_variables():
+        if name.startswith(incomplete):
+            completion_item = (name, help_text)
+            yield completion_item
```

Comment:
Nit: for brevity I would personally forego the additional binding and just do `yield (name, help_text)`